frontend/pages/hosts/details/HostDetailsPage/modals/SelectQueryModal/SelectQueryModal.tsx

```diff
@@ -88,10 +88,9 @@ const SelectQueryModal = ({
     if (!queriesFilter && queriesCount === 0) {
       return (
         <div className={`${baseClass}__no-queries`}>
-          <span className="info__header">You have no saved queries.</span>
-          <span className="info__data">
-            Expecting to see queries? Try again in a few seconds as the system
-            catches up.
+          <span className="info__header">
+            You have no saved queries. Expecting to see queries? Try again in a
+            few seconds as the system catches up.
           </span>
```

Comment:
Text is visually identical; separate spans are not necessary.

Reply:
I'm wondering if it should be similar to the other empty states where they are separated and somehow the css for info__header and info__data changed?

Reply:
Product question (cc: @noahtalerman) One big block? Or separate text?

Reply:
> similar to the other empty states

@RachelElysia similar to other empty (no results) states makes sense to me! In these states I think we have a header and description on 2 lines.

cc @lukeheath

Reply:
👍  Updated styles to match.